### PR TITLE
feat(tokens): add global and theme 

### PR DESCRIPTION
**Pull request by kamleshchandnani**

### CODE REVIEW

File: packages/blade/tokens/global/colors.ts [hunk 1]
```diff
@@ -46,9 +46,11 @@ type color = Readonly<{
     sapphire: colorChromaticScale;
   };
   neutral: {
-    blueGray: colorNeutralScale;
+    blueGrayLight: colorNeutralScale;
+    blueGrayDark: colorNeutralScale;
+    navyGrayLight: colorNeutralScale;
+    navyGrayDark: colorNeutralScale;
     warmGray: colorNeutralScale;
```
Comment (saurav12) [inline packages/blade/tokens/global/colors.ts:53]: Convert `warmGray` to `ashGrayLight`.

File: packages/blade/tokens/global/colors.ts [hunk 2]
```diff
@@ -46,9 +46,11 @@ type color = Readonly<{
...
-    navyGray: colorNeutralScale;
     coolGray: colorNeutralScale;
```
Comment (saurav12) [inline packages/blade/tokens/global/colors.ts:54]: Convert `coolGray` to `ashGrayDark`.

File: packages/blade/tokens/global/colors.ts [hunk 3]
```diff
@@ -220,8 +239,25 @@ const colors: color = {
       1200: `hsla(231, 59%, 13%, ${opacity[9]})`,
       1300: `hsla(231, 67%, 10%, ${opacity[9]})`,
     },
+    warmGray: {
```
Comment (saurav12) [inline packages/blade/tokens/global/colors.ts:242]: Convert `warmGray` to `ashGrayLight`.

File: packages/blade/tokens/global/colors.ts [hunk 4]
```diff
@@ -220,8 +239,25 @@ const colors: color = {
...
+      600: `hsla(215, 6%, 60%, ${opacity[9]})`,
+      700: `hsla(214, 6%, 55%, ${opacity[9]})`,
+      800: `hsla(216, 7%, 45%, ${opacity[9]})`,
+      900: `hsla(218, 9%, 36%, ${opacity[9]})`,
+      1000: `hsla(219, 12%, 28%, ${opacity[9]})`,
+      1100: `hsla(214, 15%, 18%, ${opacity[9]})`,
+      1200: `hsla(216, 15, 13%, ${opacity[9]})`,
+      1300: `hsla(214, 24%, 6%, ${opacity[9]})`,
+    },
     coolGray: {
```
Comment (saurav12) [inline packages/blade/tokens/global/colors.ts:259]: Convert `coolGray` to `ashGrayDark`.

File: packages/blade/tokens/theme/neoTheme.ts [hunk 5]
```diff
@@ -0,0 +1,340 @@
...
+        },
+        700: {
+          onLight: globalColors.chromatic.azure[700],
+          onDark: globalColors.chromatic.azure[600],
+        },
+      },
+      secondary: {
+        500: {
+          onLight: globalColors.chromatic.emerald[500],
+          onDark: globalColors.chromatic.emerald[500],
```
Comment (saurav12) [inline packages/blade/tokens/theme/neoTheme.ts:31]: Change values of `secondary.500` to `cider[600]` for **neoTheme**,
```
onLight: globalColors.chromatic.cider[600],
onDark: globalColors.chromatic.cider[600],
```

**Review by saurav12 — CHANGES_REQUESTED** [review]
Everything I've pointed has been replicated on Figma as well. Please update here as well.

File: .eslintrc.js [hunk 6]
```diff
@@ -6,6 +6,7 @@ module.exports = {
     'max-lines-per-function': 'off',
     'no-console': ['warn'],
     complexity: ['off'],
+    'object-property-newline': ['error', { allowAllPropertiesOnSameLine: false }],
```
Comment (dhruvdutt) [inline .eslintrc.js:9]: Strangely this is not already existing in kent's or prettier's recommended config. We should add it to universe config as well.

**Review by dhruvdutt — APPROVED** [review]

**Review by chaitanyadeorukhkar — APPROVED** [review]

**Pull request merged**